packages/dds/tree/docs/undo/v1-undo-example-flow.md

```diff
@@ -0,0 +1,269 @@
+# V1 Undo Example Flow
+
+**Legend**:
+
+```{mermaid}
```

Comment:
I think these need to be `mermaid` and not `{mermaid}` for the diagrams to embed properly ([github's docs](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams)). When I view the rich diff I'm seeing the "code" for the diagram instead of the diagrams themselves.